src/test/java/uk/gov/hmcts/reform/managecase/service/noc/RequestNoticeOfChangeServiceTest.java

```diff
@@ -316,4 +324,131 @@ private void updateCaseResourceData(CaseResource caseResource,
 
         caseResource.setData(data);
     }
+
+    @Nested
+    @DisplayName("Set Organisation To Remove")
+    @SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.JUnitTestsShouldIncludeAssert", "PMD.ExcessiveImports"})
+    class SetOrganisationToRemove {
+
+        private Organisation organisation;
+        private OrganisationPolicy organisationPolicy;
+        private ChangeOrganisationRequest changeOrganisationRequestBefore;
+        private ChangeOrganisationRequest changeOrganisationRequestAfter;
+        private CaseDetails callbackCaseDetails;
+
+        @BeforeEach
+        void setUp() {
+            organisation = Organisation.builder()
+                .organisationID("Org1")
+                .organisationName("Organisation 1")
+                .build();
+
+            organisationPolicy = OrganisationPolicy.builder()
+                .organisation(organisation)
+                .orgPolicyReference("PolicyRef")
+                .orgPolicyCaseAssignedRole("Role1")
+                .build();
+
+            changeOrganisationRequestBefore = ChangeOrganisationRequest.builder()
+                .organisationToAdd(new Organisation("123", "Org1"))
```

Comment:
Can you use builder to avoid need of changes when OrgName is removed? Likewise elsewhere in this and other test scripts.

Reply:
i.e. `Organisation.builder().organisationID("123").build()` instead of two field constructor would help when OrgName is retired.  Repeat for other occurrences in your test code.